tools/build.js

```diff
@@ -94,6 +97,22 @@ async function vendorDeno() {
   console.log(`📦 Vendoring Deno modules into '${vendorDest}'...`)
 
   await execaCommand(`deno vendor ${vendorSource} --output=${vendorDest} --force`)
+
+  // htmlrewriter contains wasm files and those don't currently work great with vendoring
+  // see https://github.com/denoland/deno/issues/14123
+  // to workaround this we copy the wasm files manually
```

Comment:
Deno landed it, but in 2.1 - https://deno.com/blog/v2.1#first-class-wasm-support , so support is not guaranteed :( otherwise we could migrate to imports in https://github.com/netlify/htmlrewriter/blob/edb477af3d08359c8f25edd8d301df69ffcc8e4b/pkg/htmlrewriter.js#L1184 to make use of it